docs/decisions/0007-release-process.md

```diff
@@ -1,6 +1,18 @@
-# Context and Problem Statement
+# Release Process
 
-Releasing is an essential process for the `s3gw` process. Given the project is
+## Note Before
+
+This document is superseded by three different ADRs, covering the same topics
+and expanding them: [Release Methodology][adr_methodology],
+[Release Steps][adr_steps], and [Release Testing][adr_testing].
+
+[adr_methodology]: /docs/decisions/0015-release-methodology.md
+[adr_steps]: /docs/decisions/0016-release-steps.md
+[adr_testing]: /docs/decisions/0017-release-testing.md
+
+## Context and Problem Statement
+
+Releasing is an essential process for the `s3gw` project. Given the project is
```

Comment:
https://github.com/aquarist-labs/s3gw/blob/c0e27c53fb42970b2c4125cd6748a639a926108d/docs/decisions/0007-release-process.md?plain=1#L22 can you please fix the hyperlink?

Reply:
Addressing that is outside the scope of this PR.